apps/shell/src/app/modules/prompt/snippets/category.tsx

```diff
@@ -0,0 +1,33 @@
+import clsx from 'clsx';
+import { useMemo } from 'react';
+
+export interface SnippetCategoryProps {
+  className?: string;
+  name: string;
+  selectedCategory: string;
+  onClick: () => void;
+}
+
+export function SnippetCategory({
+  name,
+  className,
+  onClick,
+  selectedCategory,
+}: SnippetCategoryProps) {
+  const classNames = useMemo(
+    () =>
+      clsx(
+        'block py-0.5 px-3 focus:outline-none hover:bg-zinc-700 w-full text-left',
+        selectedCategory === name && 'font-bold',
+        className,
```

Comment:
I think you can use the object syntax for readability here.

```suggestion
        className,
        {
          'font-bold': selectedCategory === name,
        },
```